build/scripts/cleanup_perf.ps1

```diff
@@ -13,25 +16,49 @@ try {
 }
 catch {}
 try {
+    taskkill /F /IM csc.exe 2>&1 | Out-Null
+}
+catch {}
+try {
+    taskkill /F /IM vbc.exe 2>&1 | Out-Null
+}
+catch {}
+try {
     taskkill /F /IM VBCSCompiler.exe 2>&1 | Out-Null
 }
 catch {}
 
-# Move all etl files to the a folder for archiving
-echo "creating ToArchive directory"
-mkdir ToArchive
-echo "moving C:/CPC/DataBackup* to ToArchive"
-mv C:/CPC/DataBackup* ToArchive
-ls ToArchive
+try {
+    & $CPCLocation/cpc.exe /Stop /SkipClean 2>&1 | Out-Null
+}
+catch {}
+
+If (Test-Path ToArchive) {
+    Remove-Item -Recurse -Force ToArchive
+}
+
+If (Test-Path $CPCLocation) {
+    # Move all etl files to the a folder for archiving
+    echo "creating ToArchive directory"
+    mkdir ToArchive
+    echo "moving C:/CPC/DataBackup* to ToArchive"
+    mv C:/CPC/DataBackup* ToArchive
+    ls ToArchive
+}
 
 # Clean CPC and related directories out of the machine
-If (Test-Path C:/CPC) {
+$ExitCode = 0
+If (Test-Path $CPCLocation) {
     try {
-        echo "removing C:/CPC ..."
-        Remove-Item -Recurse -Force C:/CPC
+        echo "removing $CPCLocation ..."
+        Remove-Item -Recurse -Force $CPCLocation
         echo "done."
     }
-    catch { }
+    catch {
+        echo "Removing CPC failed, restarting the machine.  THIS SHOULD NOT HAPPEN.  Please email tyoverby@microsoft.com"
+        shutdown /r /t 5
```

Comment:
These scripts are only supposed to be used on Jenkins.  Local runs don't go through these.